Tests/TestReferenceTypes/FixtureReferenceTypes.swift

```diff
@@ -0,0 +1,130 @@
+import MockoloFramework
+
+let referenceTypesWithWeakAnnotation = """
```

Comment:
Change names to modifier--. Also file name s/FixtureReferenceTypes/FixtureModifiers.